docs/release/release_series.md

```diff
@@ -55,6 +65,21 @@ Since the start of the RPM-based OSG software stack, we have offered the followi
     Historically, there were 3.0.x releases as well, but there was no separate release series for 3.0 and 3.1;
     we simply went from 3.0.10 to 3.1.0 in the same repositories.
 
+Series Life-cycle
+-----------------
+
+Support ends at the end of the month of the following dates unless otherwise specified:
+
+| Release Series | Initial Release | End of Regular Support | End of Critical Bug/Security Support |
+|:--------------:|-----------------|------------------------|--------------------------------------|
+| 3.6            | Februrary 2021  | Not set                | Not set                              |
+| 3.5            | August 2019     | August 2021            | 1 May 2022                           |
+| 3.4            | June 2017       | February 2020          | November 2020                        |
+| 3.3            | August 2015     | December 2017          | May 2018                             |
+| 3.2            | November 2013   | February 2016          | August 2016                          |
+| 3.1            | April 2012      | October 2014           | April 2015                           |
```

Comment:
```suggestion
| 3.5            | August 2019     | 1 August 2021            | 1 May 2022                           |
| 3.4            | June 2017       | 1 February 2020          | 1 November 2020                        |
| 3.3            | August 2015     | 1 December 2017          | 1 May 2018                             |
| 3.2            | November 2013   | 1 February 2016          | 1 August 2016                          |
| 3.1            | April 2012      | 1 October 2014           | 1 April 2015                           |
```
For consistency. Would be nice if we had the days for the initial release as well but that's less important.

Alternatively you could change "1 May 2022" to just "May 2022" for the 3.5 end of critical support date, but it looks weird to have some of them filled in and some not.

Reply:
Shouldn't those all be the end of the month? We added 1 May 2022 for the 3.5 EOL because it was out of the ordinary

Reply:
Either way, let's just be consistent in the formatting.

Reply:
Done